types/src/front/api_handlers/internal/agent_configuration.ts

```diff
@@ -111,6 +111,50 @@ export const PostOrPatchAgentConfigurationRequestBodySchema = t.type({
               })
             ),
           }),
+          t.type({
```

Comment:
this is seriously getting out of hand :/

Reply:
Yep will be factored once we create a base class for actions